File: Csharp/bitkyShop/bitkyShop/bean/Commodity.cs
using System.Collections.Generic;
using cn.bmob.io;

namespace bitkyShop.bean
{
    public class Commodity : BmobTable
    {
        public string Category { set; get; }
        public string CategorySub { set; get; }
        public string Name { set; get; }
        public BmobDouble Price { set; get; }
        public BmobInt Count { set; get; }
        public string Details { set; get; }
        public string Promotion { set; get; }
        public string AD { set; get; }
        public string BitkyId { set; get; }
        public BmobInt BitkyMode { set; get; }
        public string BitkyModeStr { set; get; }
        public string CoverPhotoUrl { set; get; }
        public List<string> CoverPhotoUrlSet { set; get; }
        public string CoverPhotoName { get; set; }
        public List<string> CoverPhotoNameSet { set; get; }

        public Commodity()
        {
            CategorySub = "";
            BitkyModeStr = "none";
            BitkyId = "default";
            BitkyMode = 0;
            CoverPhotoUrlSet = new List<string>();
            CoverPhotoNameSet = new List<string>();
        }

        public Commodity SetStamp(string msg)
        {
            BitkyId = msg;
            return this;
        }

        public Commodity SetMode(int msg)
        {
            BitkyMode = msg;
            return this;
        }

        //读字段信息
        public override void readFields(BmobInput input)
        {
            base.readFields(input);

            BitkyId = input.getString("BitkyId");
            BitkyMode = input.getInt("BitkyMode");
            BitkyModeStr = input.getString("BitkyModeStr");
            Promotion = input.getString("Promotion");
            AD = input.getString("AD");
            Category = input.getString("Category");
            CategorySub = input.getString("CategorySub");
            Name = input.getString("Name");
            Price = input.getDouble("Price");
            Count = input.getInt("Count");
            Details = input.getString("Details");
            CoverPhotoName = input.getString("CoverPhotoName");
            CoverPhotoUrl = input.getString("CoverPhotoUrl");
            CoverPhotoNameSet = input.getList<string>("CoverPhotoNameSet");
            CoverPhotoUrlSet = input.getList<string>("CoverPhotoUrlSet");
        }

        //写字段信息
        public override void write(BmobOutput output, bool all)
        {
            base.write(output, all);

            output.Put("BitkyId", BitkyId);
            output.Put("BitkyMode", BitkyMode);
            output.Put("BitkyModeStr", BitkyModeStr);
            output.Put("Promotion", Promotion);
            output.Put("AD", AD);
            output.Put("Category", Category);
            output.Put("CategorySub", CategorySub);
            output.Put("Name", Name);
            output.Put("Price", Price);
            output.Put("Count", Count);
            output.Put("Details", Details);
            output.Put("CoverPhotoName", CoverPhotoName);
            output.Put("CoverPhotoUrl", CoverPhotoUrl);
            output.Put("CoverPhotoNameSet", CoverPhotoNameSet);
            output.Put("CoverPhotoUrlSet", CoverPhotoUrlSet);
        }
    }
}

File: Android/bitkyShop/app/src/main/java/cc/bitky/bitkyshop/bean/Commodity.java
package cc.bitky.bitkyshop.bean;

import cn.bmob.v3.BmobObject;
import java.util.List;

public class Commodity extends BmobObject {
  private String BitkyId;
  private Integer BitkyMode;
  private String BitkyModeStr;
  private String Promotion;
  private String AD;
  private String Category;
  private String CategorySub;
  private String Name;
  private Double Price;
  private Integer Count;
  private String Details;
  private String CoverPhotoUrl;
  private List<String> CoverPhotoUrlSet;
  private String CoverPhotoName;
  private List<String> CoverPhotoNameSet;

  public String getPromotion() {
    return Promotion;
  }

  public void setPromotion(String promotion) {
    Promotion = promotion;
  }

  public String getAD() {
    return AD;
  }

  public void setAD(String AD) {
    this.AD = AD;
  }

  public String getCategorySub() {
    return CategorySub;
  }

  public void setCategorySub(String categorySub) {
    CategorySub = categorySub;
  }

  public String getBitkyModeStr() {
    return BitkyModeStr;
  }

  public void setBitkyModeStr(String bitkyModeStr) {
    BitkyModeStr = bitkyModeStr;
  }

  public String getBitkyId() {
    return BitkyId;
  }

  public void setBitkyId(String bitkyId) {
    BitkyId = bitkyId;
  }

  public Integer getBitkyMode() {
    return BitkyMode;
  }

  public void setBitkyMode(Integer bitkyMode) {
    BitkyMode = bitkyMode;
  }

  public String getCategory() {
    return Category;
  }

  public void setCategory(String category) {
    Category = category;
  }

  public String getName() {
    return Name;
  }

  public void setName(String name) {
    Name = name;
  }

  public Double getPrice() {
    return Price;
  }

  public void setPrice(Double price) {
    Price = price;
  }

  public Integer getCount() {
    return Count;
  }

  public void setCount(Integer count) {
    Count = count;
  }

  public String getDetails() {
    return Details;
  }

  public void setDetails(String details) {
    Details = details;
  }

  public String getCoverPhotoUrl() {
    return CoverPhotoUrl;
  }

  public void setCoverPhotoUrl(String coverPhotoUrl) {
    CoverPhotoUrl = coverPhotoUrl;
  }

  public List<String> getCoverPhotoUrlSet() {
    return CoverPhotoUrlSet;
  }

  public void setCoverPhotoUrlSet(List<String> coverPhotoUrlSet) {
    CoverPhotoUrlSet = coverPhotoUrlSet;
  }

  public String getCoverPhotoName() {
    return CoverPhotoName;
  }

  public void setCoverPhotoName(String coverPhotoName) {
    CoverPhotoName = coverPhotoName;
  }

  public List<String> getCoverPhotoNameSet() {
    return CoverPhotoNameSet;
  }

  public void setCoverPhotoNameSet(List<String> coverPhotoNameSet) {
    CoverPhotoNameSet = coverPhotoNameSet;
  }
}
